core/geometry/src/curve/Query/PlanarSubdivision.ts

```diff
@@ -23,6 +24,26 @@ import { RegionGroupMember, RegionGroupOpType } from "../RegionOpsClassification
  * @module Curve
  */
 
+function sortAngle(curve: CurvePrimitive, fraction: number, reverse: boolean): number {
```

Comment:
I understand that you just moved this function but the name is confusing. Initially I thought it sorts some angles but then realized that `sortAngle` is the angle used for sort-around-vertex. I think a name like `getSortAngle` or `calculateSortAngle` makes more sense (next 2 functions also start with `get`).

Reply:
Good point. Fixed.